README.md

```diff
@@ -54,6 +54,14 @@ filter!(raster_index, last_return)
 ```
 Filters are done in-place and create a new index matching the condition. It does not change the loaded dataset.
 
+Filtering can also be done compared to a computed surface.
+For example, if we want to select all points within some tolerance of the median raster from above:
```

Comment:
The raster is below, we have to think how to reword/move it to have an understandable flow.